docs/topics/intellij-idea-plugin-reviewing-issues.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * docs/intellij-idea-plugin-guide/master.adoc
+
+[id="intellij-idea-plugin-reviewing-issues_{context}"]
+= Reviewing issues
+
+You can use the {ProductShortName} plugin icons to prioritize issues based on their severity. You can  see which issues have a _Quick Fix_ automatic code replacement and which do not.
+
+The results of an analysis are displayed in a directory format, showing the _hints_ and _classifications_ for each application analyzed.
+
+A hint is a read-only snippet of code that contains a single issue that either must or should be addressed before an application can modernized or migrated. Often a suggested Quick Fix code replacement is suggested. You can accept or ignore any suggested Quick Fixes.
```

Comment:
Lot of passive voice in this sentence.

```suggestion
A _hint_ is a read-only snippet of code that contains a single issue that you should or must address before you can  modernize or migrate an application. Often the plugin suggests a Quick Fix, which you can either accept or ignore.
```

Reply:
I've incorporated your recommendations except for in the last sentence. You suggest using an active verb with the system, which violates the rule against anthropomorphism.